File-Formats/XlsIO/Improving-Performance.md

```diff
@@ -335,3 +335,16 @@ validation.EndUpdate()
 {% endhighlight %}
 {% endtabs %}  
 
+## Performance and Memory benchmark
+
+###Performance
+
+The performance of opening excel files with XlsIO with various types of data.
```

Comment:
1. The benchmarks can be further split into Reading and Creating Excel documents